RevenueCat.xcodeproj/project.pbxproj

```diff
@@ -4420,6 +4422,15 @@
 			path = PurchaseHistory;
 			sourceTree = "<group>";
 		};
+		574CBC832D944D7100B8B9FA /* Actions */ = {
+			isa = PBXGroup;
+			children = (
+				574CBC812D944D7100B8B9FA /* CustomerCenter+PreferenceKeys.swift */,
+				574CBC822D944D7100B8B9FA /* CustomerCenterView+Actions.swift */,
+			);
+			path = Actions;
+			sourceTree = "<group>";
+		};
```

Comment:
Just to be sure. If you created the folder with Xcode 16, it could be failing Xcode 15. To be safe, you could run the full test suite

Reply:
All good 🚀